stdlib/3/builtins.pyi

```diff
@@ -96,6 +103,10 @@ class type:
     def mro(self) -> List[type]: ...
     def __instancecheck__(self, instance: Any) -> bool: ...
     def __subclasscheck__(self, subclass: type) -> bool: ...
+    # there is a better way to specify the return type. It must
+    # be subscriptable, but it may not be a Mapping/MutableMapping.
```

Comment:
Why can't it be a `Mapping`? It actually returns a `dict`, which is a Mapping.

Reply:
I guess I read that as "it is not allowed to be a Mapping", which is not what you meant. :)

If we need to be really precise, we could use a Protocol (I suppose it needs to return something that implements `__getitem__`, `__setitem__`, and `__delitem__`.) But I think `MutableMapping` is fine.

Reply:
I have used Mapping now - I think that's more precise. It doesn't need to be mutable. Let me know if that's ok.